README.md

```diff
@@ -70,6 +70,26 @@ module "observability_agents" {
 }
 ```
 
+### Customise Kubernetes metadata line filtering in the LogDNA agent
+
+This module allows to customise the Kubernetes metadata line filtering in the LogDNA agent by setting the value of the input parameters `logdna_agent_custom_line_exclusion` and `logdna_agent_custom_line_inclusion`. For more information, see [Configuration for Kubernetes Metadata Filtering](https://github.com/logdna/logdna-agent-v2/blob/3.8/docs/KUBERNETES.md#configuration-for-kubernetes-metadata-filtering)
+
+For example, you can set the following input values for the parameters:
+
+```text
+custom_logdna_at_agent_line_exclusion = "label.app.kubernetes.io/name:sample-app\\, annotation.user:sample-user"
+custom_logdna_at_agent_line_inclusion = "namespace:default"
+```
+
+The resulting daemonset configuration will be:
+
+```text
+- name: LOGDNA_K8S_METADATA_LINE_INCLUSION
+    value: "label.app.kubernetes.io/name:sample-app, annotation.user:sample-user"
+- name: LOGDNA_K8S_METADATA_LINE_EXCLUSION
+    value: "namespace:default"
```

Comment:
the `value` here looks like it is indented too much. Should be inline with `name`